### ISSUE TITLE
VAE Training Details

### ISSUE THREAD

**Issue by ahmed1996said** [microsoft/MedImaging-ModelDriftMonitoring#21]
Dear Authors,

Thanks for the great work, and for making the repository public!

I'm training my own VAE on PadChest dataset, and I was wondering if you can provide the hyperparameters used in training? Can you confirm whether or not the hyperparameters listed on [this notebook](https://github.com/microsoft/MedImaging-ModelDriftMonitoring/blob/main/notebooks/setup/train_vae.ipynb) are correct? In the paper, it was mentioned that both frontal and lateral images were used in VAE training, but in the notebook I can see these args: 
```   
'frontal_only': 1,
'val_frontal_only': 0,
'ignore_nonfrontal_loss': 1,
```
Also, can you provide any details about the final performance of the VAE (SSIM, Reconstruction Loss, etc.) so I can ensure I'm reproducing the model correctly?

Thanks in advance!
